src/main/fc/rc_modes.h

```diff
@@ -79,6 +79,7 @@ typedef enum {
     BOXBEEPERMUTE,
     BOXREADY,
     BOXLAPTIMERRESET,
+    BOXSERVOCUSTOMRATE,
```

Comment:
Another single-use box (but that is more conceptual problem of BF and not this PR)